README.md

```diff
@@ -100,13 +100,15 @@ Once the script has finished, you will be able to access Dataverse via:
 - [http://localhost:8000/spa](http://localhost:8000/spa): SPA Frontend
 - [http://localhost:8000](http://localhost:8000): Dataverse Backend and JSF Frontend
 
-If you want to run the environment with aggregated test data (collections and datasets), run the following command:
+_Note: The Dataverse configbaker takes some time to start the application, so the application will not be accessible until the bootstrapping is complete._
```

Comment:
I ran the script without errors but I can only see the Payara home page running on localhost:8000, is there any additional step?

Reply:
Unfortunately we have to wait a few minutes until the configbaker bootstrapping process is finished. Please let me know if this time is excessive on your localhost or you find other kind of issues.

Reply:
I tried again and it worked this time without waiting too much. Thanks!